pynxtools/dataconverter/helpers.py

```diff
@@ -591,7 +592,7 @@ def add_default_root_attributes(data, filename):
     data["/@file_name"] = filename
     data["/@file_time"] = str(datetime.now(timezone.utc).astimezone())
     data["/@file_update_time"] = data["/@file_time"]
-    data["/@NeXus_version"] = "NOT_IMPLEMENTED"
+    data["/@NeXus_version"] = get_nexus_version()
```

Comment:
is not it overwritten by writer?

Reply:
No, this is the attribute in NXroot. The other one is the `/entry/definition` path.

Reply:
Currently, the writer is able to overwrite this without notice. I would

1. Swap the population of the writer and the NXroot general arguments, that way the general attributes always overwrite the reader ones.
2. Emit a warning if this field is already present there